McCalib/src/McCalib.cpp

```diff
@@ -117,15 +117,16 @@ Calibration::Calibration(const std::string config_path) {
   std::map<int, cv::Ptr<cv::aruco::CharucoBoard>> charuco_boards;
   int offset_count = 0;
   for (int i = 0; i <= max_board_idx; i++) {
-    cv::Ptr<cv::aruco::CharucoBoard> charuco = cv::aruco::CharucoBoard::create(
-        number_x_square_per_board_[i], number_y_square_per_board_[i],
+    cv::Ptr<cv::aruco::CharucoBoard> charuco = new cv::aruco::CharucoBoard(
+        cv::Size(number_x_square_per_board_[i], number_y_square_per_board_[i]),
         length_square, length_marker, dict_);
+
     if (i != 0) // If it is the first board then just use the standard idx
     {
-      int id_offset = charuco_boards[i - 1]->ids.size() + offset_count;
+      int id_offset = charuco_boards[i - 1]->getIds().size() + offset_count;
       offset_count = id_offset;
-      for (auto &id : charuco->ids)
-        id += id_offset;
+      // for (int &id : charuco.getIds())
+      //   id += id_offset;
     }
```

Comment:
Same comment as https://github.com/rameau-fr/MC-Calib/pull/80#discussion_r1739301792 @rameau-fr

Reply:
Fixed in a33577ce13e7165e54536689600b0f38d8bbd8ac